bisqapps/shared/domain/src/commonMain/kotlin/network/bisq/mobile/domain/offerbook/market/MarketListItem.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+package network.bisq.mobile.domain.offerbook.market
+
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.serialization.Serializable
+import network.bisq.mobile.client.replicated_model.common.currency.Market
+
+/**
+ * Provides data for offerbook market list items
+ */
+@Serializable
+class MarketListItem(val market: Market) {
```

Comment:
Yes it's a view model. As it is filled in the services we cannot use the presentation module as there is no dependency to that. I will move the serviceFacades to a new service package inside domain, so that it is better separated and move the view models into data/model.

Reply:
I leave the package structure as it is now, as I assume there will be more classes later and it keeps it better organized IMO if not all at the same level. If it turns out its not growing much we can move it one level lower to avoid  packages with only one class inside...